CMakeLists.txt

```diff
@@ -10,6 +10,10 @@ macro(build_uncrustify)
     list(APPEND extra_cmake_args -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE})
   endif()
   if(DEFINED CMAKE_TOOLCHAIN_FILE)
+    if(QNX)
+      # defined in toolchain file
+      set_qnx_external_project_options()
```

Comment:
Where is this function defined? Can you please reference to see its content.

Reply:
https://github.com/asobhy-qnx/ros2_root/blob/1719bcfa11a71e854fc8a76a1031fff2d4a0047a/platform/qnx.nto.toolchain.cmake#L106-L118

This is a temporary place for this file. We need to think of the best location of where to keep it.